src/S3/MultipartCopy.php

```diff
@@ -52,21 +54,22 @@ class MultipartCopy extends AbstractUploadManager
      *   result of executing a HeadObject command on the copy source.
      *
      * @param S3ClientInterface $client Client used for the upload.
-     * @param string $source Location of the data to be copied
-     *                                  (in the form /<bucket>/<key>).
+     * @param string $source Location of the data to be copied (in the form
+     *                       /<bucket>/<key>).  If the key contains a '?'
+     *                       character, instead pass an array of source_key,
+     *                       source_buckey, and source_version_id.
```

Comment:
"source_buckey" should be "source_bucket"

Shouldn't the type for this `@param` be updated to reflect `string|array`?

Reply:
good catches :) thank you